docs/source/metrics_reference.rst

```diff
@@ -6,324 +6,324 @@ Prometheus Metrics
-+-----------------------------------------------------+-----------+------------------------------------------------------------+------------------+-------------------------------------------------------------+
-| ledger_blockstorage_and_pvtdata_commit_time         | histogram | Time taken in seconds for committing the block and private | channel          |                                                             |
-|                                                     |           | data to storage.                                           |                  |                                                             |
-+-----------------------------------------------------+-----------+------------------------------------------------------------+------------------+-------------------------------------------------------------+
-| ledger_blockstorage_commit_time                     | histogram | Time taken in seconds for committing the block to storage. | channel          |                                                             |
-+-----------------------------------------------------+-----------+------------------------------------------------------------+------------------+-------------------------------------------------------------+
-| ledger_statedb_commit_time                          | histogram | Time taken in seconds for committing block changes to      | channel          |                                                             |
-|                                                     |           | state db.                                                  |                  |                                                             |
-+-----------------------------------------------------+-----------+------------------------------------------------------------+------------------+-------------------------------------------------------------+
-| ledger_transaction_count                            | counter   | Number of transactions processed.                          | channel          |                                                             |
-|                                                     |           |                                                            +------------------+-------------------------------------------------------------+
-|                                                     |           |                                                            | transaction_type |                                                             |
-|                                                     |           |                                                            +------------------+-------------------------------------------------------------+
-|                                                     |           |                                                            | chaincode        |                                                             |
-|   
```

Comment:
If I am reading this correctly, it looks like some of the metrics may not have a Source? Is that intentional?

Reply:
"Source" is a misnomer and part of why the request is not really trivial. Metrics are in namespaces and are simply emitted. Some of these emissions come from the peer, some from the order, some from both. There are others that are specific to the CA and not documented here.